piecrust/src/instance.rs

```diff
@@ -151,13 +151,13 @@ impl WrappedInstance {
         let instance = wasmer::Instance::new(&mut store, &module, &imports)?;
 
         // Ensure there is a global exported named `A`, whose value is in the
-        // first few pages of memory.
```

Comment:
Curiosity: why this has changed?

Reply:
The contract *can technically* put it's argument buffer wherever it wants in its address space. Since we consider all space to be allocated - even if its not really - the argument buffer can be anywhere.